lib/web_ui/lib/src/engine/compositor/canvaskit_api.dart

```diff
@@ -1629,3 +1675,39 @@ external Object? get _finalizationRegistryConstructor;
 
 /// Whether the current browser supports `FinalizationRegistry`.
 bool browserSupportsFinalizationRegistry = _finalizationRegistryConstructor != null;
+
+class SkData {
```

Comment:
This needs to be annotated with `@JS()`

Reply:
Sure. Used to be before the Big Reorganization. :-)